app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -28,7 +28,6 @@ import { CommitOneLine } from '../../models/commit'
 import { PopupType } from '../../models/popup'
 import { RepositorySettingsTab } from '../repository-settings/repository-settings'
 import { isRepositoryWithForkedGitHubRepository } from '../../models/repository'
-import { debounce } from 'lodash'
```

Comment:
Just curious: why not using `debounce` anymore? 🤔

Reply:
`debounce` just doesn't work properly for async functions like this, it's not designed to handle promises. I did a bunch of testing where I forced a manual delay, and it never worked as long as I was using `debounce`. [This issue](https://github.com/lodash/lodash/issues/4400) was raised on the Lodash repo as well.

Reply:
Oh, interesting! Thanks for the context 💖